plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/StringPathUtil.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import java.util.regex.Pattern;
+
+public final class StringPathUtil
+{
+    public static final String PATH_SEPARATOR = "/";
+    public static final Pattern DOUBLE_SLASH_TO_REPLACE = Pattern.compile("(?<!\\w+:/?)//+");
```

Comment:
I need a few unit tests to see whether there is no cheaper alternative to this negative look behind regex.